Gopkg.toml

```diff
@@ -46,10 +46,6 @@
   name = "github.com/go-sql-driver/mysql"
   version = "1.4.0"
 
-[[constraint]]
```

Comment:
Don't forget to run `dep ensure` prior to committing the gopkg.lock changes

Reply:
Hmm.  Not sure why when I run this on `fdamador\telegraf`, I get errors related to `influxdata\telegraf` references.

> Subpackage github.com/influxdata/telegraf/*/* is missing. (Package is required by (root).)